### PR TITLE
Implemented Login Screen with Reusable Components

### LINKED ISSUE #5: Log in screen
## REQUIREMENTS / DESCRIPTION

Create a log in screen that allows use the email and password, also have option to log in using a google accound, also having a link to register screen

## DESIGN
:)

### PR DESCRIPTION
**Pull request by Tole15**
This commit introduces the Login Screen, which has been developed using a reusable component approach. By leveraging reusable components, we have improved code efficiency, enhanced design consistency, and ensured easier maintenance of the application.

The Login Screen incorporates various reusable components that contribute to its overall design and functionality. These components have been carefully crafted to serve specific purposes, resulting in a cohesive and user-friendly login experience.

By employing a reusable component strategy, we have achieved a more streamlined development process, allowing for faster iteration and updates in the future. This approach also promotes code reusability and ensures a consistent design language across different screens of the application.
Screenshot_1688110376

### CODE REVIEW

**Review by DerekAyala — APPROVED** [review]
LGTM!

**Review by montsegv-2 — APPROVED** [review]
LGTM, good job!

File: src/components/LogInForm.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const LogInForm = () => {
+  const {
+    email,
+    setEmail,
+    password,
+    setPassword,
+    termsChecked,
+    setTermsChecked,
+    subscribeChecked,
+    setSubscribeChecked,
```
Comment (Eduardoespl) [inline src/components/LogInForm.jsx]: U need this lines in this file?

**Review by Eduardoespl — APPROVED** [review]
LGTM!

**Review by Ewar22 — APPROVED** [review]
Nice

**Review by GustavoHiguera — APPROVED** [review]
LGTM

**Pull request merged**